pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundleFactory.java

```diff
@@ -159,14 +159,16 @@ private NamespaceBundles readBundles(NamespaceName namespace, byte[] value, long
 
     private void handleMetadataStoreNotification(Notification n) {
         if (n.getPath().startsWith(LOCAL_POLICIES_ROOT)) {
-            final NamespaceName namespace = NamespaceName.get(getNamespaceFromPoliciesPath(n.getPath()));
```

Comment:
here, aren't we expecting some well known path ?
how this is possible ?

Reply:
n.getPath() may return "/admin/local-policies" or "/admin/local-policies/aaa" that not satisfied with 6 slashes.
getNamespaceFromPoliciesPath(n.getPath()) may return "".  and cause NamespaceName.get() throw IllegalArgumentException.